src/property-observation.js

```diff
@@ -96,7 +96,7 @@ export class SetterObserver {
 
     if (!Reflect.defineProperty(this.obj, this.propertyName, {
       configurable: true,
-      enumerable: true,
+      enumerable: this.obj.propertyIsEnumerable(this.propertyName),
```

Comment:
It might be better to replace this with `this.propertyName in this.obj ? this.obj.propertyIsEnumerable(this.propertyName) : true`. But I'm not sure whether the property is always existent.